content/all-spaces/disconnected-spaces/helm-reference.md

```diff
@@ -274,7 +287,10 @@ This reference provides detailed documentation on the Upbound Space Helm chart.
 | features.alpha.gitSource | object | | None |
 | features.alpha.gitSource.enabled | bool | `true` | None |
 | features.alpha.kine | object | | None |
-| features.alpha.kine.enabled | bool | `false` | None |
+| features.alpha.kine.accountSchema.enabled | bool | `false` | None |
+| features.alpha.kine.enabled | bool | `false` | This configures kine to store its data in a separate schema for each account, and configures apollo to query it. If disabled, kine will store all data in the public schema. |
+| features.alpha.kine.upboundRBAC | bool | `false` | None |
```

Comment:
If one is running a high enough version of Spaces, it enables authorization based on `authorization.spaces.upbound.io` API types.

Reply:
What is the minimum version number?